### DISCUSSION TITLE
The way how the ip would fetched should changed

### DISCUSSION THREAD

**Discussion by scysys** [AzuraCast/AzuraCast#6161] [General]
Under "Streamers," there are "Connection Informations" displayed. That's fine and working, but there is a scenario where it become a display problem.

If the server has both activated IPv4 and IPv6, AzuraCast will fetch the IP that will be displayed from here: https://central.azuracast.com/ip

When i check the following command on those servers:

`curl https://central.azuracast.com/ip`

You'll get by default the V6 address

`{"ip":"2001:41d0:701:xxxx::xxxx"}`

This is not optimal, as it will display to a user: "You may need to connect directly via your IP address: 2001:41d0:701:xxxx::xxxx." However, if this user does not have an IPv6-ready connection, these display informations will be wrong and just not working.

I would change it myself, but as I see the codebase on central.azuracast.com is not open. So I would prefer you to change the output there to first get the IPv4 address instead of IPv6. This will work in all scenarios.

**Comment by Sella-GH** [comment]
I found something out about this, maybe it was changed in the last few months.
If I use this command on the server I get the correct IPv6 address returned but AzuraCast still connects via the IPv4 one as seen in the streamer connection info screen.

Can this behavior maybe be changed so it listens on both IP addresses instead of one? And then prefers the v6 over the v4?

**Reply by BusterNeece** [reply]
@Sella-3DX Well, I think you have the opposite opinion of the original poster here. We switched to using IPV4 only when responding with IP addresses because most users would likely not understand IPV6 addresses (or be in an environment that only partially supported them), which is why you don't see IPV6 addresses from that API endpoint.
